src/sections/Community/Handbook/repo-data.js

```diff
@@ -58,6 +58,16 @@ export const repo_data = [
         language: "Cytoscape.js",
         repository: "https://github.com/meshery/meshery",
       },
+      {
+        project: "Layer5 discussion forum",
+        image: five,
+        site: "https://discuss.layer5.io/",
+      },
+      {
+        project: "layer5.io/learn-ng",
+        image: five,
+        site: "layer5.io/learn-ng",
```

Comment:
add link to github  and language

Reply:
I couldn't find the links , can you send if you have the links.

Reply:
@YashKamboj learn-ng is a part of layer5 site itself, I don't think we need to specify this separately here. You can remove this from the table altogether.